docs/coding-guidelines/adding-api-guidelines.md

```diff
@@ -4,15 +4,12 @@ Recommended reading to better understand this document:
 | [Package-Projects](package-projects.md)
 
 # Add APIs
```

Comment:
Fix [MD022](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines).

```suggestion
# Add APIs

```